content/en/docs/guides/certificate_management/certificate-rotation.md

```diff
@@ -0,0 +1,197 @@
+
+Run `kubectl get meshrootcertificate` to see the default `MeshRootCertificate`:
+
+```yaml
+apiVersion: config.openservicemesh.io/v1alpha2
+kind: MeshRootCertificate
+metadata:
+  name: osm-mesh-root-certificate
+  namespace: osm-system
+spec:
+  role: active
+  provider:
+    tresor:
+      ca:
+        secretRef:
+          name: osm-ca-bundle
+          namespace: osm-system
+  spiffeEnabled: false
+  trustDomain: cluster.local
+```
+
+The only field that can be modified after initial creation is the `role` field.  Other fields will need change by creating a new MeshRootCertificate and go through the [rotation process](#meshrootcertificate-rotation). On a new install, the initial `MeshRootCertificate` will be created by OSM and the certificate will be in the **active** `role`. Fields such as `trustDomain` and `spiffeEnabled` can be configured by passing options to the install command:
+
+```
+osm install --set="osm.featureFlags.enableSPIFFE=true --set"osm.trustDomain=cluster.example"
+```
+
+### MeshRootCertificate Rotation
+
+To rotate the root certificate a mesh, a second `MeshRootCertificate` (MRC) will be created.  The second `MeshRootCertificate` will be created in a `passive` role and rotated into the `active` role. Once the new certificate is `active`, the old certificate will be moved to the `passive` role then finally `inactive`.  
+
+The full process for rotating the certificate is:
+
+| Step  |  original MRC(1) role   |  new MRC(2) role    | Signing MRC  | Validating MRC |
+| ----- | ----------------------- | ------------------- | ------------ | -------------- |
+| 1     | active                  | (not created)       | mrc1         | mcr1           |
+| 2     | active                  | passive             | mcr1         | mcr1 and mrc2  |
+| 3     | active                  | active              | mrc1 or mrc2 | mcr1 and mrc2  |
+| 4     | passive                 | active              | mcr2         | mcr1 and mrc2  |
+| 5     | inactive                | active              | mcr2         | mcr2           |
+| 5     | (deleted)               | active              | mcr2         | mcr2           |
+
+
+Each step requires time for the certificate information to propagated to all the components in the mesh. The amount of time required will depend on your mesh. 
+
+Between each step to verify a certificate was been propagated you can review the logs in osm-controller and optionally run the follow commands. When both certificates are in use, then you will see two certificates from the following commands, otherwise you will see the `active` certificate.  We are working on improving this experience as we move the `MeshRootCertificate` feature out of preview.
```

Comment:
```suggestion
To verify if a certificate has been propagated, you can review the OSM controller logs and optionally run the following commands. If both certificates are in use, then you will see two certificates from the following commands, otherwise you will see the `active` certificate.  

We are working on improving this experience as we move the `MeshRootCertificate` feature out of preview.
```